src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -596,6 +609,9 @@ const WorkflowView = _.flow(
     const inputsValid = _.isEmpty(errors.inputs)
     const outputsValid = _.isEmpty(errors.outputs)
     const sourceDisplay = sourceRepo === 'agora' ? `${methodNamespace}/${methodName}/${methodVersion}` : `${methodPath}:${methodVersion}`
+    const checkBoxSpanMargins = { margin: '0 0.5rem 0 1rem' }
+    const checkBoxLeftMargin = { marginLeft: '1rem' }
```

Comment:
nit: could you move these styles to the styles object on line 51?

Reply:
Thanks for pointing me to the styles object, done.